{{cookiecutter.project_name}}/.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -23,67 +23,74 @@ body:
-        ipykernel           6.15.0
-        jedi                0.18.1
-        mpl_toolkits        NA
-        natsort             8.1.0
-        numpy               1.22.4
-        packaging           21.3
-        pandas              1.4.2
-        parso               0.8.3
-        pexpect             4.8.0
-        pickleshare         0.7.5
-        pkg_resources       NA
-        prompt_toolkit      3.0.29
-        psutil              5.9.1
-        ptyprocess          0.7.0
-        pure_eval           0.2.2
-        pydev_ipython       NA
-        pydevconsole        NA
-        pydevd              2.8.0
-        pydevd_file_utils   NA
-        pydevd_plugins      NA
-        pydevd_tracing      NA
-        pygments            2.12.0
-        pytz                2022.1
-        scipy               1.8.1
-        setuptools          62.5.0
-        setuptools_scm      NA
-        six                 1.16.0
-        stack_data          0.3.0
-        tornado             6.1
-        traitlets           5.3.0
-        wcwidth             0.2.5
-        zmq                 23.1.0
-        -----
-        IPython             8.4.0
-        jupyter_client      7.3.4
-        jupyter_core        4.10.0
-        -----
-        Python 3.9.13 | packaged by conda-forge | (main, May 27 2022, 16:58:50) [GCC 10.3.0]
-        Linux-5.18.6-arch1-1-x86_64-with-glibc2.35
-        -----
-        Session information updated at 2022-07-07 17:55
+        anndata	0.11.3
+        ----	----
+        charset-normalizer	3.4.1
+        coverage	7.7.0
+        psutil	7.0.0
+        dask	2024.7.1
+        jaraco.context	5.3.0
+        numcodecs	0.15.1
+        jaraco.functools	4.0.1
+        Jinja2	3.1.6
+        sphinxcontrib-jsmath	1.0.1
+        sphinxcontrib-htmlhelp	2.1.0
+        toolz	1.0.0
+        session-info2	0.1.2
+        PyYAML	6.0.2
+        llvmlite	0.44.0
+        scipy	1.15.2
+        pandas	2.2.3
+        sphinxcontrib-devhelp	2.0.0
+        h5py	3.13.0
+        tblib	3.0.0
+        setuptools-scm	8.2.0
+        more-itertools	10.3.0
+        msgpack	1.1.0
+        sparse	0.15.5
+        wrapt	1.17.2
+        jaraco.collections	5.1.0
+        numba	0.61.0
+        pyarrow	19.0.1
+        pytz	2025.1
+        MarkupSafe	3.0.2
+        crc32c	2.7.1
+        sphinxcontrib-qthelp	2.0.0
+        sphinxcontrib-serializinghtml	2.0.0
+        zarr	2.18.4
+        asciitree	0.3.3
+        six	1.17.0
+        sphinxcontrib-applehelp	2.0.0
+        numpy	2.1.3
+        cloudpickle	3.1.1
+        sphinxcontrib-bibtex	2.6.3
+        natsort	8.4.0
+        jaraco.text	3.12.1
+        setuptools	76.1.0
+        Deprecated	1.2.18
+        packaging	24.2
+        python-dateutil	2.9.0.post0
+        ----	----
+        Python	3.13.2 | packaged by conda-forge | (main, Feb 17 2025, 14:10:22) [GCC 13.3.0]
+        OS	Linux-6.11.0-109019-tuxedo-x86_64-with-glibc2.39
+        Updated	2025-03-18 15:47
+    validations:
+      required: true
```

Comment:
You really want to force this? I'd still rather have users to submit incomplete bug reports than not reporting them at all.